Srclib.Nuget/ConsoleCommands/DepresolveConsoleCommand.cs

```diff
@@ -10,124 +10,135 @@
 
 namespace Srclib.Nuget
 {
-  class DepresolveConsoleCommand
-  {
-    static Lazy<string> _dnuPath;
-
-    public static void Register(CommandLineApplication cmdApp, Microsoft.Extensions.PlatformAbstractions.IApplicationEnvironment appEnv, Microsoft.Extensions.PlatformAbstractions.IRuntimeEnvironment runtimeEnv)
+    class DepresolveConsoleCommand
     {
-      if (runtimeEnv.OperatingSystem == "Windows")
-      {
-        _dnuPath = new Lazy<string>(FindDnuWindows);
-      }
-      else
-      {
-        _dnuPath = new Lazy<string>(FindDnuNix);
-      }
-
-      cmdApp.Command("depresolve", c => {
-        c.Description = "Perform a combination of parsing, static analysis, semantic analysis, and type inference";
-
-        c.HelpOption("-?|-h|--help");
-
-        c.OnExecute(async () => {
-          var jsonIn = await Console.In.ReadToEndAsync();
-          var sourceUnit = JsonConvert.DeserializeObject<SourceUnit>(jsonIn);
-
-          var dir = Path.Combine(Directory.GetCurrentDirectory(), sourceUnit.Dir);
-          var deps = await DepResolve(dir);
-
-          var result = new List<Resolution>();
-          foreach(var dep in deps)
-          {
-            result.Add(Resolution.FromLibrary(dep));
-          }
-
-          Console.WriteLine(JsonConvert.SerializeObject(result, Formatting.Indented));
-          return 0;
-        });
-      });
-    }
+        static Lazy<string> _dnuPath;
```

Comment:
If we assume dnu is on the system path, anyway (which we do, because the way we compute this value is by doing `which dnu` or `where dnu`), then why is it even necessary to get the absolute path? Can't we just call `dnu` and it will work because it's on the system path?